spec/paths/reports@subscription-cancellation.yaml

```diff
@@ -0,0 +1,54 @@
+get:
+  tags:
+    - Reports
+  summary: Retrieve transactions with their plan data
+  description: Retrieve transactions with their plan data
+  parameters:
+    - name: periodStart
+      in: query
+      description: Report start date
+      type: string
+      required: true
+    - name: periodEnd
+      in: query
+      description: Report end date
+      type: string
+      required: true
+    - name: aggregationField
+      in: query
+      description: Report aggregation field
+      required: true
+      type: string
+      enum:
+        - canceled-by
+        - cancel-category
+        - leads.affiliate
```

Comment:
Yes, there is property field. But I reviewed other resources and found out that in other places the field names are in camel case. So I've decided to change this.